js/mathCalc.js

```diff
@@ -0,0 +1,559 @@
+function getCrusader(id) { return jsonData.crusaders.find(function(c){ return c.id == id; }); } 
+
+
+function itemDPS(rarity) {
+  switch(rarity){
+    case 1:
+      return 1.05;
+    case 2:
+      return 1.10;
+    case 3:
+      return 1.15;
+    case 4:
+      return 1.4;
+    case "Golden Epic":
+      return 1.6;
+    case 5:
+      return 1.8;
+    case "Golden Legendary":
+      return 2.2;
+    default:
+      return 1;
+  }
+}
+
+function itemSelfDPS(rarity) {
+  switch(rarity){
+    case 1:
+      return 1.25;
+    case 2:
+      return 1.5;
+    case 3:
+      return 2;
+    case 4:
+      return 5;
+    case "Golden Epic":
+      return 7;
+    case 5:
+      return 9;
+    case "Golden Legendary":
+      return 13;    
+    default:
+      return 1;
+  }
+}
+
+function itemGold(rarity) {
+  switch(rarity){
```

Comment:
Is the point of moving these to lower the load on the clientside calculation? It would make sense to do it for that, but seems like a lot of repetitive data entry since it is just based on the rarity level rather than being specific to the item itself.